src/messages/proposals.rs

```diff
@@ -182,17 +207,42 @@ impl ProposalQueue {
     }
     /// Returns a new `ProposalQueue` from proposals that were committed and
```

Comment:
That comment is not accurate here, is it? Shouldn't these be proposals that are not committed and do need filtering?

Reply:
This just means they are not filtered now.